akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -1058,7 +1058,7 @@ private[akka] class Shard(
    * of active entities.
    */
   @InternalStableApi
-  def entityCreated(@unused id: EntityId): Int = entities.activeEntities().size
+  def entityCreated(@unused id: EntityId): Int = entities.size
```

Comment:
This would be a change in behaviour for Cinnamon from before we re-did remembered entities. This could include entities scheduled to be started by remembered entities which would be misleading in the stats  as we throttle the starting.

Agreed that original impl is too expensive to be called every time an entity is created.

Instead we could add a `def nrActiveEntities() = byRef.size` to the Entities class

Reply:
Yes, `byRef` size should be used. Updated, please check.